src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -165,90 +183,103 @@ module.exports = function<T, P, I, C>(config : HostConfig<T, P, I, C>) {
     } while (child = child.sibling);
   }
 
+  function bailoutOnCurrent(current : Fiber, workInProgress : Fiber) : ?Fiber {
+    // The most likely scenario is that the previous copy of the tree contains
+    // the same props as the new one. In that case, we can just copy the output
```

Comment:
Yea, we verify this at the callsite of this function. The "most likely" part is just saying that we should we check for that first before doing anything else.